e2e/cypress/tests/support/api/bots.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {ChainableT} from 'tests/types';
+
+import {getRandomId} from '../../utils';
+
+// *****************************************************************************
+// Bots
+// https://api.mattermost.com/#tag/bots
+// *****************************************************************************
+
+interface ApiCreateBotArg {
+    prefix?: string;
+    bot?: Cypress.BotPatch;
+}
+
+function apiCreateBot({prefix, bot = createBotPatch(prefix)}: ApiCreateBotArg): ChainableT<any> {
```

Comment:
Found error running the test. Please take a look.
<img width="916" alt="Screenshot 2023-01-13 at 12 44 27 PM" src="https://user-images.githubusercontent.com/5334504/212239551-4e964c6c-d16a-4749-a708-924483fed52b.png">

This would be nice to change the signature. Something like:
```javascript
function apiCreateBot(prefix = 'bot'): ChainableT<any> {
    const bot = createBotPatch(prefix);

    ...
}
```